src/types/common.ts

```diff
@@ -2,4 +2,20 @@ export enum PageComponent {
   Index = 'index',
   Slack = 'slack',
   CodeOfConduct = 'code-of-conduct',
-}
\ No newline at end of file
+}
+
+export interface Event {
```

Comment:
We have a type for this. Could you unify it and use a single type throughout the app? Feel free to remove mine, as I added it to a component.